tests/client/helpers.js

```diff
@@ -71,7 +71,7 @@ export function unexpectedSuccess() {
 /*
  * Creates a stand-in for a jed instance,
  */
-export function getFakeI18nInst() {
+export function getFakeI18nInst({ lang = 'en-US' } = {}) {
```

Comment:
I know this is only a test, but do we declare `en-US` as the default locale anywhere else? Would be nice to use a constant rather than the magic one here.